addons/mail/static/src/discuss/core/web/discuss_sidebar_categories.xml

```diff
@@ -49,15 +49,15 @@
             }"
             t-on-click="(ev) => this.openThread(ev, thread)"
         >
-            <div class="bg-inherit position-relative d-flex ms-4 flex-shrink-0" style="width:30px;height:30px">
+            <div class="bg-inherit position-relative d-flex ms-3 flex-shrink-0" style="width:26px;height:26px">
```

Comment:
Why do we have style inline instead of defining a class?

Reply:
Probably because this is more straight-forward